src/core/gltf2importer/gltf2parser_p.h

```diff
@@ -127,17 +139,36 @@ class KUESA_PRIVATE_EXPORT GLTF2Parser
     Qt3DCore::QEntity *scene(const int id);
 
     QString m_basePath;
-    GLTF2Context *m_context;
+    GLTF2Context *m_context = nullptr;
```

Comment:
can we keep all initialisation in one place? Either here or in the constructor (I prefer the later)